packages/model/src/research-output.ts

```diff
@@ -118,6 +120,11 @@ export const isResearchOutputType = (
 ): type is ResearchOutputType =>
   (researchOutputTypes as ReadonlyArray<string>).includes(type);
 
+export const isResearchOutputPublishingEntity = (
```

Comment:
is this used anywhere outside of this file? if not why export it - i see little value in testing this unit of code, it's so little logic i would just move it into the `researchOutputMapPublishingEntity` function, that is, if it's the only place it is actually used in